tests/run-make/track-pgo-dep-info/rmake.rs

```diff
@@ -0,0 +1,22 @@
+// Emitting dep-info files used to not have any mention of PGO profiles used
+// in compilation, which meant these profiles could be changed without consequence.
+// After changing this in #100801, this test checks that the profile data is successfully
+// included in dep-info emit files.
+// See https://github.com/rust-lang/rust/pull/100801
+
+//FIXME(Oneirical): try it on musl
+//@ needs-profiler-support
+
+use run_make_support::{fs_wrapper, llvm_profdata, run, rustc};
+
+fn main() {
+    // Generate the profile-guided-optimization (PGO) profiles
+    rustc().profile_generate("profiles").input("main.rs").run();
+    // Merge the profiles
+    run("main");
```

Comment:
Question: does this need `//@ ignore-cross-compile`? Or is that kinda implied by `//@ needs-profiler-support` (even if it is, maybe having the ignore is clearer?)?

Reply:
I believe that it's indeed getting implied by //@ needs-profiler-support, but I added the ignore + reason for clarity.